docs/source/io_formats/fission_energy.rst

```diff
@@ -0,0 +1,52 @@
+.. _usersguide_fission_energy:
+
+==================================
+Fission Energy Release File Format
+==================================
+
+This file is a compact HDF5 representation of the ENDF MT=1, MF=458 data (see
+ENDF-102_ for details).  It gives the information needed to compute the energy
+carried away from fission reactions by each reaction product (e.g. fragment
+nuclei, neutrons) which depends on the incident neutron energy.  OpenMC is
+distributed with one of these files under
+data/fission_Q_data_endfb71.h5.  More files of this format can be created from
+ENDF files with the
+``openmc.data.write_compact_458_library`` function.  They can be read with the
+``openmc.data.FissionEnergyRelease.from_compact_hdf5`` class method.
+
+:Attributes: - **comment** (*char[]*) -- An optional text comment
+             - **component order** (*char[][]*) -- An array of strings
+               specifying the order each reaction product occurs in the data
+               arrays.  The components use the 2-3 letter abbreviations
+               specified in ENDF-102 e.g. EFR for fission fragments and ENP for
+               prompt neutrons.
+
+**/<nuclide name>/**
+    Nuclides are named by concatenating their atomic symbol and mass number. For
+    example, 'U235' or 'Pu239'.  Metastable nuclides are appended with an
+    '_m' and their metastable number.  For example, 'Am242_m1'
+
+:Datasets: - **data** (*double[][][]*) -- The energy release coefficients.  The
```

Comment:
Can you make the following change here:
```diff
diff --git a/docs/source/io_formats/fission_energy.rst b/docs/source/io_formats/fission_energy.rst
index 768db56..0b21a84 100644
--- a/docs/source/io_formats/fission_energy.rst
+++ b/docs/source/io_formats/fission_energy.rst
@@ -26,7 +26,8 @@ ENDF files with the
     example, 'U235' or 'Pu239'.  Metastable nuclides are appended with an
     '_m' and their metastable number.  For example, 'Am242_m1'
 
-:Datasets: - **data** (*double[][][]*) -- The energy release coefficients.  The
+:Datasets:
+           - **data** (*double[][][]*) -- The energy release coefficients.  The
              first axis indexes the component type.  The second axis specifies
              values or uncertainties.  The third axis indexes the polynomial
              order.  If the data uses the Sher-Beck format, then the last axis
```
Sphinx/rst is kind of weird. It expects a blank line before and after a bullet list, but I really only get errors when its a bullet list with only one item.